README
+=============================================================================+
|                               Animal Crossing                               |
|                               Bell Calculator                               |
+=============================================================================+
| This is a way to find out how many bells you will get from selling bugs and |
| fish to the Nooklings in Animal Crossing: New Horizons! This is no more than|
| a glorified multiplication calculator.                                      |
+=============================================================================+
|                                  Features:                                  |
|-----------------------------------------------------------------------------|
| - Can search up any fish                                                    |
| - Can search up any bug                                                     |
+=============================================================================+
|                                  Planned:                                   |
|-----------------------------------------------------------------------------|
| - Implement a way to add multiple bugs/fish and not just one                |
| - Add all Animal Crossing games in one file                                 |
+=============================================================================+


CHANGELOG
*******************************************************************************
*                                    ACBC                                     *
*                                  Changelog                                  *
*******************************************************************************
* v0.2:                                                                       *
*       - Added Support for CJ and Flick                                      *
*                                                                             *
* v0.1:                                                                       *
*       - Initial Release                                                     *
*******************************************************************************

fish.lua
-- This is a list of all fish in Animal Crossing: new Horizons. I can probably slim it down due to duplicate amounts, but I see no real benifet to it.
local price ={
    ['Anchovy'] = 200,
    ['Angelfish'] = 3000,
    ['Arapaima'] = 10000,
    ['Arowana'] = 10000,
    ['Barred Knifejaw'] = 5000,
    ['Barreleye'] = 15000,
    ['Betta'] = 2500,
    ['Bitterling'] = 900,
    ['Black Bass'] = 400,
    ['Blowfish'] = 5000,
    ['Blue Marlin'] = 10000,
    ['Bluegill'] = 180,
    ['Butterfly Fish'] = 1000,
    ['Carp'] = 300,
    ['Catfish'] = 800,
    ['Char'] = 3800,
    ['Cherry Salmon'] = 1000,
    ['Clown Fish'] = 650,
    ['Coelacanth'] = 15000,
    ['Crawfish'] = 200,
    ['Crucian Carp'] = 160,
    ['Dab'] = 300,
    ['Dace'] = 240,
    ['Dorado'] = 15000,
    ['Football Fish'] = 2500,
    ['Freshwater Goby'] = 400,
    ['Frog'] = 120,
    ['Gar'] = 6000,
    ['Giant Snakehead'] = 5500,
    ['Giant Trevally'] = 4500,
    ['Golden Trout'] = 15000,
    ['Goldfish'] = 1300,
    ['Great White Shark'] = 15000,
    ['Guppy'] = 1300,
    ['Hammerhead Shark'] = 8000,
    ['Horse Mackerel'] = 150,
    ['Killifish'] = 300,
    ['King Salmon'] = 1800,
    ['Koi'] = 4000,
    ['Loach'] = 400,
    ['Mahi-Mahi'] = 6000,
    ['Mitten Crab'] = 2000,
    ['Moray Eel'] = 2000,
    ['Napoleonfish'] = 10000,
    ['Neon Tetra'] = 500,
    ['Nibble Fish'] = 1500,
    ['Oarfish'] = 9000,
    ['Ocean Sunfish'] = 4000,
    ['Olive Flounder'] = 800,
    ['Pale Chub'] = 200,
    ['Pike'] = 1800,
    ['Piranha'] = 2500,
    ['Pond Smelt'] = 500,
    ['Pop-Eyed Goldfish'] = 1300,
    ['Puffer Fish'] = 250,
    ['Rainbowfish'] = 800,
    ['Ranchu Goldfish'] = 4500,
    ['Ray'] = 3000,
    ['Red Snapper'] = 3000,
    ['Ribbon Eel'] = 600,
    ['Saddled Bichir'] = 4000,
    ['Salmon'] = 700,
    ['Saw Shark'] = 12000,
    ['Sea Bass'] = 400,
    ['Sea Butterfly'] = 1000,
    ['Sea Horse'] = 1100,
    ['Snapping Turtle'] = 5000,
    ['Soft-Shelled Turtle'] = 3750,
    ['Squid'] = 500,
    ['Stringfish'] = 15000,
    ['Sturgeon'] = 10000,
    ['Suckerfish'] = 1500,
    ['Surgeonfish'] = 1000,
    ['Sweetfish'] = 900,
    ['Tadpole'] = 100,
    ['Tilapia'] = 800,
    ['Tuna'] = 7000,
    ['Whale Shark'] = 13000,
    ['Yellow Perch'] = 300,
    ['Zebra Turkeyfish'] = 500
}

--[[
    This is for adding CJ.

local enable ={
    ['Yes'] = Yes,
    ['No'] = No
}
]]

-- Just a welcome message that always shows when you start the application.
print("Animal Crossing Bell Calculator v0.2")
print("Hello and thank you for using the Animal Crossing Bell Calculator! Please view \nthe README to see what all you can do with this application. If you'd like to \nsee what has been added since v0.1 please view the file named CHANGELOG.\n\n")

-- Asks how many fish you are selling and stores it as variable `num1`
print("How many fish are you selling?")
local num1 = tonumber(io.read()) -- tonumber() makes the number not a string.
print()

--[[
    TODO: Implement CJ modifier correctly!!!

print("Are you selling to CJ? [Yes/No]")
local cj = enable[io.read()]
print()
]]

-- Asks what fish you are selling and stores it as `fish`
print("What kind of fish are you selling?")
local fish = price[io.read()]
print()

-- Multiplies the number of fish being sold and the fish from the table above!
print("You will get " .. num1*fish .. " for selling to the nooklings!\n") -- Use .. instead of , so it is joined by spaces, not tabs!!!
print("You will get " .. num1*2*fish .. " for selling to CJ!\n")

os.exit()

bugs.lua
-- This is a list of all the bugs in Animal Crossing: New Horizons.
local prices ={
    ['Agrias Butterfly'] = 3000,
    ['Ant'] = 80,
    ['Atlas Moth'] = 3000,
    ['Bagworm'] = 600,
    ['Banded Dragonfly'] = 4500,
    ['Bell Cricket'] = 430,
    ['Blue Weevil Beetle'] = 800,
    ['Brown Cicada'] = 250,
    ['Centipede'] = 300,
    ['Cicada Shell'] = 10,
    ['Citrus Long-Horned Beetle'] = 350,
    ['Common Bluebottle'] = 300,
    ['Common Butterfly'] = 160,
    ['Cricket'] = 130,
    ['Cyclommatus Stag'] = 8000,
    ['Damselfly'] = 500,
    ['Darner Dragonfly'] = 230,
    ['Diving Beetle'] = 800,
    ['Drone Beetle'] = 200,
    ['Dung Beetle'] = 3000,
    ['Earth-Boring Dung Beetle'] = 300,
    ['Emperor Butterfly'] = 4000,
    ['Evening Cicada'] = 550,
    ['Firefly'] = 300,
    ['Flea'] = 70,
    ['Fly'] = 60,
    ['Giant Cicada'] = 500,
    ['Giant Stag'] = 10000,
    ['Giant Water Bug'] = 2000,
    ['Giraffe Stag'] = 12000,
    ['Golden Stag'] = 12000,
    ['Goliath Beetle'] = 8000,
    ['Grasshopper'] = 160,
    ['Great Purple Emperor'] = 3000,
    ['Hermit Crab'] = 1000,
    ['Honeybee'] = 200,
    ['Horned Atlas'] = 8000,
    ['Horned Dynastid'] = 1350,
    ['Horned Elephant'] = 8000,
    ['Horned Hercules'] = 12000,
    ['Jewel Beetle'] = 2400,
    ['Ladybug'] = 200,
    ['Long Locust'] = 200,
    ['Madagascan Sunset Moth'] = 2500,
    ['Man-Faced Stink Bug'] = 1000,
    ['Mantis'] = 430,
    ['Migratory Locust'] = 600,
    ['Miyama Stag'] = 1000,
    ['Mole Cricket'] = 500,
    ['Monarch Butterfly'] = 140,
    ['Mosquito'] = 130,
    ['Moth'] = 130,
    ['Orchid Mantis'] = 2400,
    ['Paper Kite Butterfly'] = 1000,
    ['Peacock Butterfly'] = 2500,
    ['Pill Bug'] = 250,
    ['Pondskater'] = 130,
    ['Queen Alexandra\'s Birdwing'] = 4000,
    ['Rainbow Stag'] = 6000,
    ['Rajah Brooke\'s Birdwing'] = 2500,
    ['Red Dragonfly'] = 180,
    ['Rice Grasshopper'] = 160,
    ['Robust Cicada'] = 300,
    ['Rosalia Batesi Beetle'] = 3000,
    ['Saw Stag'] = 2000,
    ['Scarab Beetle'] = 10000,
    ['Scorpion'] = 8000,
    ['Snail'] = 250,
    ['Spider'] = 600,
    ['Stinkbug'] = 120,
    ['Tarantula'] = 8000,
    ['Tiger Beetle'] = 1500,
    ['Tiger Butterfly'] = 240,
    ['Violin Beetle'] = 450,
    ['Walker Cicada'] = 400,
    ['Walking Leaf'] = 600,
    ['Walking Stick'] = 600,
    ['Wasp'] = 2500,
    ['Warf Roach'] = 200,
    ['Yellow Butterfly'] = 160
}

--[[
    This is for adding Flick.

local enable ={
    ['Yes'] = Yes,
    ['No'] = No
}
]]

-- Welcome message that is displayed everytime you run the application.
print("Animal Crossing Bell Calculator v0.2")
print("Hello and thank you for using the Animal Crossing Bell Calculator! Please view \nthe README to see what all you can do with this application. If you'd like to \nsee what has been added since v0.1 please view the file named CHANGELOG.\n\n")

-- Asks user how many bugs they are selling.
print("How many bugs are you selling?")
local num1 = tonumber(io.read()) -- tonumber() makes the number not a string.
print()

--[[
    TODO: Implement this code to add Flick modifier!!!

print("Are you selling to Flick? [Yes/No]")
local flick = enable[io.read()]
print()
]]

-- Asks user what bug they are selling.
print("What bug are you selling?")
local bug = prices[io.read()]
print()

--[[
    Prints out the amount of bells you will recieve!
    TODO: Have if statement so it doesn't print out all of the output at once!!!!
]]
print("You will get " .. num1*bug .. " bells for selling to the nooklings!\n") -- Use .. instead of , so it is joined by spaces, not tabs!!!
print("You will get " .. num1*2*bug .. " bells for selling to Flick!\n")

os.exit()